content/swiftui/concepts/notifications/notifications.md

```diff
@@ -0,0 +1,202 @@
+---
+Title: 'Notifications'
+Description: 'Learn how to use notifications in SwiftUI for event handling and communication.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'Notifications'
+Table of Contents:
+- Introduction
+- Alerts
+- Action Sheets
+- Toasts (Message Overlays)
+- Custom Notifications
+- Best Practices
+- Conclusion
+---
+# SwiftUI Notifications
+
+## Introduction 
+
+In **[SwiftUI](https://developer.apple.com/documentation/swiftui/)**, **`notifications`** refer to visual and interactive user interface components that display messages, alerts, or updates to the user. 
+
+They are used to provide information, prompt user interaction, or display important messages within your app's user interface. 
+
+SwiftUI provides several types of notifications that we can use to enhance the user experience. We can use modifiers and view modifiers to create and disply notifications as part of the interface. 
+
+If in Swift (using NotificationCenter) notifications are primarly a mechanism for communication between different app components, in SwiftUI notifications are visual elements used to present information or gather inputs from the user. 
```

Comment:
```suggestion
In Swift (using NotificationCenter), notifications are primarily a mechanism for communication between different app components. They are also visual elements used to present information or gather inputs from the user. 
```

Reply:
Thank you for all you comments and suggestions. I applied them all.